Tests/FoundationExtensionsTests/MutableParameterTests.swift

```diff
@@ -0,0 +1,168 @@
+// Copyright © 2023 Lautsprecher Teufel GmbH. All rights reserved.
+
+import FoundationExtensions
+import XCTest
+
+class MutableParameterTests: XCTestCase {
+    func testMutableParameterRequestChangeTo20WhenValue15ChangingTo20() {
+        // given
+        let input: MutableParameter<Int> = .value(15)
+
+        // when
+        let sut = input.requestChange(to: 20)
+
+        // then
+        let result: MutableParameter<Int> = .changing(old: 15, new: 20)
+
+        XCTAssertEqual(sut, result)
+    }
+
+    func testMutableParameterRequestChangeMutatingTo20WhenValue15ChangingTo20() {
+        // given
+        var input: MutableParameter<Int> = .value(15)
+
+        // when
+        input.requestChangeMutating(to: 20)
+
+        // then
+        let result: MutableParameter<Int> = .changing(old: 15, new: 20)
+
+        XCTAssertEqual(input, result)
+    }
+
+    func testMutableParameterSuccessRequestWhenValue15Nil() {
+        // given
+        let input: MutableParameter<Int> = .value(15)
+
+        // when
+        let sut = input.successRequest()
+
+        // then
+        let result: MutableParameter<Int>? = nil
+
+        XCTAssertEqual(sut, result)
+    }
+
+    func testMutableParameterSuccessRequestWhenChangingFrom15To20Value20() {
```

Comment:
Was the name supposed to be `testMutableParameterSuccessRequestWhenChangingFrom15ToValue20`?

Reply:
I like this convention and underscores would improve the readability a lot.